### PR TITLE
Overlay additional functionality

### PR DESCRIPTION
**Pull request by beneblubb**
Overlay class should work as intended.
Remaining Problem: Not possible to toggle Visibility by pressing a button once. So it toggles the visibility 60 times a second until you release the key. Makes it kind of random.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -79,13 +82,17 @@ def create_overlays(ground, ego):
...
 
 act_change_box_color_on_spacebar_pressed = LandingGameActionOnKey(
     PygameKeyState(pygame.K_SPACE, True), color_key_indicator_blue
 )
 act_change_box_color_on_spacebar_not_pressed = LandingGameActionOnKey(
     PygameKeyState(pygame.K_SPACE, False), color_key_indicator_cyan
 )
+act_toggle_overlays_on_v_pressed = LandingGameActionOnKey(
+    PygameKeyState(pygame.K_v, True), lambda: toggle_overlays(overlays)
+)
```
Comment (beneblubb) [inline main.py:95]: Is it better to merge it like this or wait until the underlying bug is fixed and adjust the code of this pr before merging?

**Comment by beneblubb** [comment]
This draft will be finished after Issue #20 is resolved.

**Review by Dummyrunner — CHANGES_REQUESTED** [review]
* clarify naming of key press events
* clarify concept of key press when hold down is not required
* create unit tests that mock key input

**Comment by beneblubb** [comment]
@Dummyrunner I fit this PR to the current state of main (with the possibility to trigger an event on pressing down a key)

**Review by Dummyrunner — APPROVED** [review]

**Pull request merged**